src/guest_webapp/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# choose node
+FROM node:20-alpine AS build_react
```

Comment:
Unless doing the multi-staged building, avoid using the "as" keyword

Reply:
Eventually, we will implement multi-stage building. Currently, we are only in the development environment, but once we set up the production environment, we’ll use:
```Dockerfile
FROM nginx:latest (or caddy:latest)
COPY --from:build_react ...
COPY ...
```
For now, we can omit `AS` and add it back when we establish the production environment.